plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -169,6 +169,57 @@ public void testTimestampLogAppendModePushDown()
         });
     }
 
+    @Test
+    public void testTimestampCreateTimeModeOpenRangePushDown()
```

Comment:
Please add missing operators `=`, `>`, `<=` and `BETWEEN`. The above change affects them.

Reply:
OK, I added these cases in test.